test/features/install_dataindex.feature

```diff
@@ -17,3 +17,29 @@ Feature: Kogito Data Index
       }
     }
     """
+
+#####
+
+  @persistence
+  @external-infinispan
```

Comment:
shouldn't we use something more general ?
it seems we have 2 information in one tag here
we could put something like:
- @externalcomponent
- @infinispan